scripts/generate-new-migration.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+echo "What is the name of the migration file you want to create (excluding version number)?"
+
+read NEW_FILENAME
+
+if ! [[ $NEW_FILENAME =~ ^[a-z]+(_[a-z]+)*$ ]]; then
+    echo "Error: Filename must be in snake_case (lowercase letters separated by underscores)"
+    exit 1
+fi
+
+## Get most recent version number
+PROJECT_ROOT="$(cd "$(dirname "$0")/.." && pwd)"
+cd "$PROJECT_ROOT/api/src/db/migrations/"
+
+MOST_RECENT_FILENAME=$(ls -1v $(pwd) | sort -V | tail -n 1)
+
+MOST_RECENT_FILENAME_WITHOUT_FILETYPE=$(echo $MOST_RECENT_FILENAME | cut -d "." -f 1)
+
+MOST_RECENT_FILE_VERSION=$(echo $MOST_RECENT_FILENAME | cut -d "_" -f 1)
+
+MOST_RECENT_VERSION_NUMBER=$(echo $MOST_RECENT_FILE_VERSION | cut -d "v" -f 2)
+
+NEW_VERSION_NUMBER=$(($MOST_RECENT_VERSION_NUMBER + 1))
+
+NEW_FILENAME_COMPLETE="v${NEW_VERSION_NUMBER}_${NEW_FILENAME}.ts"
+
+## Create new file
+cp "$MOST_RECENT_FILENAME" "$NEW_FILENAME_COMPLETE"
+echo "New migration created at: api/src/db/migrations/$NEW_FILENAME_COMPLETE"
+
+# Replace version numbers
+sed -i.bak "s/$MOST_RECENT_VERSION_NUMBER/$NEW_VERSION_NUMBER/g" "$NEW_FILENAME_COMPLETE"
```

Comment:
We initially using `sed -i ''` but it turns out that this flag works a little differently between Mac and Linux and isn't a cross-platform approach. This solution was found [here](https://stackoverflow.com/a/22084103/5469511).